### PR TITLE
Development

### PR DESCRIPTION
**Pull request by kofodym**
A JS Rock Paper Scissors Game without User Interface

### CODE REVIEW

File: resources/js/RockPaperScissors_Game.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    return "rock";
+  } else if (userValue === paperValue) {
+    console.log(username + " you picked, paper");
+    return "paper";
+  } else if (userValue === scissorsValue) {
+    console.log(username + " you picked, scissors ");
+    return "scissors";
+  }
+}
+
```
Comment (Phildeoner) [inline resources/js/RockPaperScissors_Game.js:39]: Leave just a single space in-between items

File: resources/js/RockPaperScissors_Game.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  ) {
+    console.log(`Computer won, ${computerInput} beats ${userInput}`);
+    countComputer++;
+    return "lose";
+  } else {
+    console.log(`INVALID, ENTER A VALID INPUT`);
+    return "lose";
+  }
+}
+
```
Comment (Phildeoner) [inline resources/js/RockPaperScissors_Game.js:75]: A single line space is ok

**Review by Phildeoner — APPROVED** [review]
Hi @kofodym 👋, you did a great job, well-done. Just go through the suggestions and proceed to merge to wrap up this project.
Happy coding😊😊

File: resources/js/RockPaperScissors_Game.js (hunk 1 above)
Comment (kofodym) [reply resources/js/RockPaperScissors_Game.js:39]: Thanks sir

**Pull request merged**